modules/installation-aws-arm-tested-machine-types.adoc

```diff
@@ -16,11 +16,11 @@ The following Amazon Web Services (AWS) 64-bit ARM instance types have been test
 
 [NOTE]
 ====
-Use the machine types included in the following charts for your AWS ARM instances. If you use an instance type that is not listed in the chart, ensure that the instance size you use matches the minimum resource requirements that are listed in "Minimum resource requirements for cluster installation". 
+Use the machine types included in the following charts for your AWS ARM instances. If you use an instance type that is not listed in the chart, ensure that the instance size you use matches the minimum resource requirements that are listed in "Minimum resource requirements for cluster installation".
 ====
 
-.Machine types based on 64-bit ARM architecture 
+.Machine types based on 64-bit ARM architecture
 [%collapsible]
 ====
-include::https://raw.githubusercontent.com/openshift/installer/master/docs/user/aws/tested_instance_types_aarch64.md[]
+include::https://raw.githubusercontent.com/openshift/installer/release-4.14/docs/user/aws/tested_instance_types_aarch64.md[]
```

Comment:
FYI https://github.com/openshift/installer/blob/release-4.14/docs/user/aws/tested_instance_types_aarch64.md also list `r8g.*`, while the preview [link](https://82043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations#installation-aws-arm-tested-machine-types_installing-aws-customizations) only lists `c6g.*` and `m6g.*`.

Reply:
LGTM for other parts, thanks!